.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: required
 dist: trusty
 language: cpp
 
-# Install Bazel.
+# Install Bazel and cppcheck.
 install:
   - sudo apt-get update
   - sudo apt-get install pkg-config zip g++ zlib1g-dev unzip python
```

Comment:
(jfyi, we use clang 6 now)

Reply:
Good to know! Which version of Thrift?

Reply:
still our internal fork of 0.9.2. :(

Reply:
In regards to compiler versions, I will expand the matrix to include more versions, both GCC and Clang. On Thrift, @mway, is the internal fork of Thrift only required for your M3 client? If so, would it be possible to replace it in one shot with this new client?

Reply:
The bigger problem @jeromefroe is the use of two versions of the same library in the same project. For example, `my_project` uses `tally-cpp` and `thrift`. The generated code for `my_project` is generated by Thrift 0.9.2. It uses the definitions from that library. `tally-cpp` needs Thrift `0.11.0`. There is no clear way to resolve the conflict other than decide on using the same version.

Reply:
Yea, my understanding is that ATG uses a monorepo internally so I was hoping that the only reason they depend on Thrift is because of their M3 client. If that's the case then I was hoping it would be possible to remove that dependency for this one atomically.

Reply:
No I heard about this for Jaeger about 6 months ago and looks like things haven't changed. They are probably using it for RPC across the board.

Reply:
Gotcha, @mway can you confirm that Thrift 0.9.2 is a hard requirement?